hub.util.exceptions.RenameError
===============================

.. currentmodule:: hub.util.exceptions

.. autoexception:: RenameError